misc/actions_stubs.py

```diff
@@ -20,7 +20,7 @@ def apply_all(
     directory: str,
     extension: str,
     to_extension: str = "",
-    exclude: Tuple[str] = ("",),
+    exclude: tuple[str] = ("",),
```

Comment:
Not related to your change but pretty sure this should be `tuple[str, ...]`.

Reply:
That looks highly plausible :) would rather not make that change in this PR, though